unit_tests/test_uda.py

```diff
@@ -85,10 +85,10 @@ def test_get_create_query_string(self):
 
     def test_load_uda_from_yaml(self):
         expected_create_string = "CREATE AGGREGATE testing.my_uda(int) " \
-                                 "SFUNC plus " \
-                                 "STYPE tinyint " \
-                                 "REDUCEFUNC plus " \
-                                 "FINALFUNC fib " \
+                                 "SFUNC xwasm_plus " \
+                                 "STYPE int " \
+                                 "REDUCEFUNC xwasm_plus " \
+                                 "FINALFUNC xwasm_div_fin " \
```

Comment:
I see that we're not actually executing this query, but this aggregate would not work because the types of `SFUNC`, `REDUCEFUNC` and `FINALFUNC` don't match.
If we wanted this UDA to be a sum, we would need the `xwasm_div_fin` to be an identity function instead, and if we wanted it to be an average, we would need to change the `SFUNC` to `xwasm_plus_acc` and the REDUCEFUNC to an UDF that sums two accumulator tuples.

Reply:
Good catch. Even if this is not executed, better for it to not be confusing. I changed it to include a `xwasm_simple_return_int` function which is just a `int` identity function.